.github/workflows/generate-ecs-mappings.yml

```diff
@@ -0,0 +1,43 @@
+name: ECS Generator
+
+on:
+  push:
+    paths:
+      - 'ecs/**/*.json'
+      - 'ecs/**/*.yml'
+
+jobs:
+  run-ecs-generator:
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Extract branch name
+        shell: bash
+        run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
+        id: branch-name
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Set up Docker Compose
+        run: sudo apt-get install docker-compose
+
+      - name: Generate PR to wazuh-indexer-plugins
+        if: github.repository == 'wazuh/wazuh-indexer'
```

Comment:
This condition should be set at job's level

```yml
jobs:
  run-ecs-generator:
    if: github.repository == 'wazuh/wazuh-indexer'
    runs-on: ubuntu-24.04
```

Reply:
Fixed on https://github.com/wazuh/wazuh-indexer/pull/586/commits/6fbfe0dfc6dc68b397de945fc803221863b41ec8